src/main/java/com/spotify/reaper/service/SchedulingManager.java

```diff
@@ -103,79 +104,131 @@ public void run() {
    * @param schedule The schedule to be checked for activation.
    * @return boolean indicating whether a new RepairRun instance was created and started.
    */
-  private boolean manageSchedule(RepairSchedule schedule) {
-    if (schedule.getNextActivation().isBeforeNow()) {
-      boolean startNewRun = true;
-      LOG.info("repair unit '{}' should be repaired based on RepairSchedule with id '{}'",
-               schedule.getRepairUnitId(), schedule.getId());
-
-      RepairUnit repairUnit = null;
-      if (schedule.getState() == RepairSchedule.State.PAUSED) {
-        LOG.info("Repair schedule '{}' is paused", schedule.getId());
-        startNewRun = false;
-      } else {
-        Optional<RepairUnit> fetchedUnit =
-            context.storage.getRepairUnit(schedule.getRepairUnitId());
-        if (!fetchedUnit.isPresent()) {
-          LOG.warn("RepairUnit with id {} not found", schedule.getRepairUnitId());
-          return false;
+  private boolean manageSchedule(RepairSchedule schedule_) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)